service/arm_templates/main.json

```diff
@@ -50,28 +50,93 @@
     "kubernetesIngressInsecurePort": {
       "type": "int"
     },
+    "masterCount": {
+      "type": "int",
+      "defaultValue": 1
+    },
+    "masterIPAddresses": {
```

Comment:
Do we have to assign the IP address? Or can Azure assign one from the masters CIDR?

In our `kubernetesd` API we have IPAM logic for assigning the subnet network ranges. But having to assign IP addresses for each node would be more difficult.

Reply:
We can dynamically assign an IP address, so I'll make it so. Was not sure if you wanted to have full control on that.

Reply:
Thanks dynamically assigning is great. In this case less control is better as otherwise the IPAM logic gets too complex in kubernetesd.